<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>994</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>569</width>
    <height>994</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>994</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>994</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Target Excel File</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="selectFile">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Select File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetFixedSize</enum>
         </property>
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldGrowthPolicy::FieldsStayAtSizeHint</enum>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="exportData">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Export Data</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="clearForm">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Clear Form</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>549</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Strategy Check List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>108</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Strategy Information</string>
           </property>
           <property name="buddy">
            <cstring>textEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>186</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="tabChangesFocus">
            <bool>false</bool>
           </property>
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::LineWrapMode::WidgetWidth</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>186</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>186</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_3">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_3">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>186</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox_4">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_4">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Strategy Selector</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Select Strategy</string>
           </property>
           <property name="buddy">
            <cstring>comboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <bold>false</bold>
            </font>
           </property>
           <property name="placeholderText">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="removeStrategy">
           <property name="text">
            <string>Remove Current Strategy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Trade Information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Date and Time of Trade</string>
         </property>
         <property name="buddy">
          <cstring>dateTimeEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Symbol</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_2</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="maximum">
          <double>999999.989999999990687</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_2">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>NQ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MNQ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MES</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Initial Investment</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_4</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="comboBox_4">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Win</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Loss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Breakeven</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Win/Loss/Breakeven</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_4</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Exit Price</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_3</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Entry Price</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Position Size</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_2</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_3">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Long</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Short</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Long/Short</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_3</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>P/L</string>
         </property>
         <property name="buddy">
          <cstring>doubleSpinBox_5</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="minimumDate">
          <date>
           <year>2020</year>
           <month>9</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
         <property name="value">
          <double>2000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelo">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInsert_Strategies"/>
    <addaction name="actionRemove_Strategy"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuHelo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionLink_Excel_Doc">
   <property name="text">
    <string>Link Excel Doc</string>
   </property>
  </action>
  <action name="actionInsert_Strategies">
   <property name="text">
    <string>Insert Strategies</string>
   </property>
  </action>
  <action name="actionRemove_Strategy">
   <property name="text">
    <string>Remove Strategy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
